src/model/SameLengthWordModel.py

```diff
@@ -0,0 +1,37 @@
+from src.classes.model_result import ModelResult
+from src.classes.prediction import Prediction
+from src.classes.text_part import TextPart
+from src.model.model import Model
+from src.model.standard_model import StandardModel
+from src.model.word_model import wordModel
+from src.utils.strings import StringUtils
+
+
+
+class SameLengthWordModel(wordModel):
+
+    def predict(self,text):
+        modelResult=super.predict(text)
```

Comment:
same here I think we need to add min_p ass well and not rely on the defult in case we would like to change it

Reply:
+1